web/src/components/dsprot/TransactionList.vue

```diff
@@ -16,4 +16,4 @@
             TransactionUI
         }
     }
-</script>  
\ No newline at end of file
+</script>
```

Comment:
```suggestion
</script>

```
missing newline at end of file